frontend/src/components/CreateUserModal/index.tsx

```diff
@@ -44,40 +54,39 @@ const textFields: TextField[] = [
   { id: 'jobTitle', name: 'Job title', type: 'text' },
   { id: 'email', name: 'Email', type: 'text' },
   { id: 'contractedHours', name: 'Contracted hours', type: 'number' },
-  { id: 'pay', name: 'Pay', type: 'number' },
+  { id: 'hourlyRate', name: 'Pay (£/hour)', type: 'number' },
 ];
 
-const CreateUserModal = ({ onClose }: Props) => {
-  const dispatch = useDispatch();
-  const { getTokenSilently } = useAuth0();
-  const [preferredDays, setPreferredDays] = useState<{
-    [key: string]: boolean;
-  }>({
-    Mon: false,
-    Tue: false,
-    Wed: false,
-    Thu: false,
-    Fri: false,
-    Sat: false,
-    Sun: false,
-  });
-
-  const [userInfo, setUserInfo] = useState<UserInfo>({
-    firstName: '',
-    surname: '',
-    jobTitle: '',
-    email: '',
-    role: RoleType.User,
-    contractedHours: 0,
-    pay: 0,
-  });
+const weekdayArray = ['Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat', 'Sun'];
+
+const defaultUserInfo: Employee = {
+  id: 0,
+  firstName: '',
+  surname: '',
+  jobTitle: '',
+  email: '',
+  role: RoleType.User,
+  contractedHours: 0,
+  hourlyRate: 0,
+  startDate: new Date(),
+  preferredDates: '0000000',
```

Comment:
For me its not the clearest representation of weekdays.

Reply:
I haven't changed this since before - this is how it's stored in the database (we couldn't agree on the best way of storing this and we didn't want to create  a separate table for preferred dates).
We could use an object `{[weekday: string]: boolean}` on the front-end, but we'll have to think how to serialize it and then process it on the back end.
This could possibly be a separate ticket?

Reply:
Ok